### PR TITLE
update travis config

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
   system_site_packages: true
 
 install:
- - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
- - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
- - travis_install_nightly 7.0
- - pip install coveralls flake8
+  - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml]: In last version of the script, you don't have to put `${VERSION}`

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 
 script:
     - travis_run_flake8
-    - travis_run_tests 7.0
+    - travis_run_tests ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml]: In last version of the script, you don't have to put `${VERSION}`

**Comment by guewen** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**